transports/rabbitmq/routing-topology.md

```diff
@@ -17,6 +17,8 @@ The RabbitMQ transport has the concept of a routing topology, which controls how
 
 partial: mandatory
 
+The `ConventionalRoutingTopology` relies on [fanout exchanges](https://www.rabbitmq.com/tutorials/amqp-concepts.html#exchange-fanout) to route messages. 
```

Comment:
I believe this will make this sentence be standalone and separate by a space from the partial content. I'd prefer to see this integrated back into the partials.